doc/source/cluster/config.rst

```diff
@@ -130,6 +132,20 @@ Provider
             :ref:`project_id <cluster-configuration-project-id>`: str
             :ref:`cache_stopped_nodes <cluster-configuration-cache-stopped-nodes>`: bool
 
+.. _cluster-configuration-security-group-type:
+
+Security Group
+~~~~~~~~~~~~~~
```

Comment:
Is this okay as a top-level header?

Reply:
I thought about this too, but saw that the current pattern is to top-level everything in `Custom types`. As an example, (`Node config`, `Node Docker`, `Resources`) are all top-level even though they're only used by `Node Types`. 

Keeping it top-level has the benefit of being able to see "everything available" in the table of contents on the right, but I'm also okay with making it hierarchical if that reads better!

Reply:
LGTM!